runtime/compilers/rillv1beta/project.go

```diff
@@ -35,7 +35,7 @@ func (c *Codec) InitEmpty(ctx context.Context, name, rillVersion string) error {
 		return err
 	}
 
-	err = c.Repo.Put(ctx, c.InstanceID, ".gitignore", strings.NewReader("*.db\n*.db.wal\ndata/\n"))
+	err = c.Repo.Append(ctx, c.InstanceID, ".gitignore", strings.NewReader("*.db\n*.db.wal\ndata/\n"))
```

Comment:
If the `.gitignore` file already exists and doesn't end with a newline, this will mess up the patterns.

We need to add a newline before, but it will look ugly when a `.gitignore` file doesn't already exist. The best approach is to conditionally add a newline before the patterns. 

Also, we should add a comment before – like `# Rill\n`.